### PR TITLE
Docs: Update Explore docs for 6.5 (time-sync button & log details)

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What this PR does / why we need it**:
Updates and adds documentation for 6.5 release for following changes:

- [x] Time-sync button https://github.com/grafana/grafana/pull/19274
- [x] UI changes for log details https://github.com/grafana/grafana/pull/20034

### CODE REVIEW

File: docs/sources/features/explore/index.md [hunk 1]
```diff
@@ -90,13 +92,16 @@ Log data can be very repetitive and Explore can help by hiding duplicate log lin
...
 
 There are some other check boxes under the logging graph apart from the Deduping options.
 
 * Timestamp: shows/hides the Timestamp column
 * Local time: shows/hides the Local time column
-* Labels: shows/hides the label filters column
+
+### Labels and Parsed fields
+
+Each log row has extendable area with its labels and parsed fields, for more robust interaction. For all labels we have added the ability to filter for (positive filter) and filter out (negative filter) selected label. Each field, as well as the label has also metrics button to display its stats in relation to all displayed logs.
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: - ...row has an extendable ....
- ...selected labels.
- Each field or label also has a stats icon to display ad-hoc statistics in relation to all displayed logs.

**Review by davkal — COMMENTED** [review]
Small tweaks, i hope you don't mind.

File: docs/sources/features/explore/index.md (hunk 1 above)
Comment (ivanahuckova) [reply docs/sources/features/explore/index.md]: updated :)

**Comment by marefr** [comment]
@ivanahuckova can you update the What's new in 6.5 article with the images you added to website repo?

**Comment by ivanahuckova** [comment]
@marefr absolutely! Can you please review and approve: https://github.com/grafana/grafana/pull/20442

**Review by aocenas — APPROVED** [review]

**Pull request merged**